Doc/whatsnew/3.12.rst

```diff
@@ -251,6 +251,10 @@ Deprecated
   :exc:`ImportWarning`).
   (Contributed by Brett Cannon in :gh:`65961`.)
 
+* :func:`pkgutil.find_module` and :func:`pkgutil.get_module`
+  now raise :exc:`DeprecationWarning`,
+  use :func:`importlib.util.find_spec` instead.
```

Comment:
```suggestion
  use :func:`importlib.util.find_spec` instead.
  (Contributed by Nikita Sobolev in :gh:`97850`.)
```

You don't have to include your name in this if you really don't want to, but it not only gives credit where it is due, but more importantly helps readers track down more information about the deprecation.